lib/wallet/balance-tracker-contract.js

```diff
@@ -6,7 +6,14 @@ class BalanceTrackerContract extends ethers.Contract {
     constructor(walletOrProvider) {
         const provider = walletOrProvider.provider || walletOrProvider;
         const networkName = provider.network.name;
-        const balanceTrackerDeployment = require(`./abis/${networkName}/BalanceTracker.json`);
+        let balanceTrackerDeployment;
+        
+        if (networkName === 'homestead') 
+            balanceTrackerDeployment = require('nahmii-contract-abstractions/build/contracts/BalanceTracker.json');
+        
+        if (networkName === 'ropsten') 
```

Comment:
I think the better behaviour here would be to just use the network name as a post-fix to the main package name (when the network is not homestead/mainnet). That way it will crash immediately with an error saying it cant find the module *nahmii-contract-abstractions-magicmushroomnetwork...* (or whatever network were trying to use)
This way it will crash where the problem is, instead of some place later.